cfme/tests/configure/test_access_control.py

```diff
@@ -32,7 +35,13 @@
 # due to pytest.mark.meta(blockers=[1035399]), non admin users can't login
 # with no providers added
 pytestmark = [test_requirements.rbac,
-              pytest.mark.usefixtures("infra_provider")]
+              pytest.mark.usefixtures("virtualcenter_provider")]
+
+
+@pytest.fixture(scope='module')
+def a_provider(request):
+    prov_filter = ProviderFilter(classes=[VMwareProvider])
```

Comment:
The same, using data stores available in vmware, but will replace with fields filter